src/igvfd/tests/test_upgrade_measurement_set.py

```diff
@@ -88,3 +88,11 @@ def test_measurement_set_upgrade_14_15(upgrader, measurement_set_v14):
     assert 'sequencing_library_types' not in value
     assert 'auxiliary_sets' not in value
     assert value['schema_version'] == '15'
+
+
+def test_measurement_set_upgrade_15_16(upgrader, measurement_set_v15):
+    value = upgrader.upgrade(
+        'measurement_set', measurement_set_v15,
+        current_version='15', target_version='16')
+    assert value['schema_version'] == '16'
```

Comment:
please add assertion of the notes update

Reply:
added